service-files/deployer.timer

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Deployer timer
+
+[Timer]
+OnCalendar=*:*:00
+Persistent=true
+
+[Install]
+WantedBy=timers.target
+
+[X-Fleet]
+MachineOf=deployer.service
+
```

Comment:
I know this works in the coco cluster as well, but how on earth will this trigger the deployer.service? I must be dumb, because I can't find any reference about it.